wpiutil/src/main/generated/java/edu/wpi/first/util/datalog/IntegerArrayLogEntry.java

```diff
@@ -2,17 +2,21 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
+// THIS FILE WAS GENERATED BY generate_datalog_entries.py DO NOT EDIT IT MANUALLY
+
 package edu.wpi.first.util.datalog;
 
+
 import java.util.Arrays;
 
-/** Log array of integer values. */
+
+/** Log int64[] values. */
 public class IntegerArrayLogEntry extends DataLogEntry {
-  /** The data type for integer array values. */
+  /** The data type for int64[] values. */
```

Comment:
Might want another field so that there's less of these changes. (For most of the other ones, it wasn't that bad, but this one is especially unhelpful)